openc3/lib/openc3/api/cmd_api.rb

```diff
@@ -314,9 +314,19 @@ def get_cmd_cnts(target_commands, scope: $openc3_scope, token: $openc3_token)
     # PRIVATE implementation details
     ###########################################################################
 
-    def cmd_implementation(method_name, *args, range_check:, hazardous_check:, raw:, timeout: nil,
+    def cmd_implementation(method_name, *args, range_check:, hazardous_check:, raw:, timeout: nil, log_message: nil,
                            scope: $openc3_scope, token: $openc3_token, **kwargs)
       extract_string_kwargs_to_args(args, kwargs)
+      unless [nil, true, false].include?(log_message)
+        raise "Invalid log_message parameter: #{log_message}. Must be true or false."
+      end
+      unless timeout.nil?
+        begin
+          Float(timeout)
+        rescue ArgumentError, TypeError
+          raise "Invalid timeout parameter: #{timeout}. Must be numeric."
+        end
+      end
```

Comment:
Added this while I was in there ... since this is an API it seemed good to check.